WultraMobileTokenSDK/Operations/Model/UserOperation/WMTTemplates.swift

```diff
@@ -0,0 +1,222 @@
+//
+// Copyright 2024 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+import Foundation
+
+/// Detailed information about displaying operation data
+///
+/// Contains prearranged styles for the operation attributes for the app to display
+public class WMTTemplates: Codable {
+    
+    /// The template how the operation should look like in the list of operations
```

Comment:
Grammar + i think `How the operation should look like in the list of operations` should be enough.